arch/loongarch/include/asm/ptrace.h

```diff
@@ -35,7 +34,11 @@ struct pt_regs {
 
 static inline int regs_irqs_disabled(struct pt_regs *regs)
 {
+#ifdef CONFIG_IRQ_PIPELINE
```

Comment:
Usually, Linux uses `ifdef` to conditionally select the function but not the statement.